### PR TITLE
chore(servicestage/auth): host parameter is no longer required

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
During the OpenAPI upgradation, the behavior of parameter host is changed from required to optional.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. update the behavior of the parameter host from required to optional.
```

## PR Checklist

* [x] Tests added/passed.

```
./scripts/coverage.sh -o servicestage -f TestAccRepoTokenAuth_basic
Prepare to calculate the coverage the following command:
TF_ACC=1 go test "./huaweicloud/services/acceptance/servicestage" -v -coverprofile="./huaweicloud/services/acceptance/servicestage/servicestage_coverage.cov" -coverpkg="./huaweicloud/services/servicestage" -run TestAccRepoTokenAuth_basic -timeout 360m -parallel 10
=== RUN   TestAccRepoTokenAuth_basic
=== PAUSE TestAccRepoTokenAuth_basic
=== CONT  TestAccRepoTokenAuth_basic
--- PASS: TestAccRepoTokenAuth_basic (10.58s)
PASS
coverage: 3.9% of statements in ./huaweicloud/services/servicestage
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/servicestage      10.630s coverage: 3.9% of statements in ./huaweicloud/services/servicestage
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

**Comment by Lance52259** [comment]
```
./scripts/coverage.sh -o servicestage -f TestAccRepoTokenAuth_basic
Prepare to calculate the coverage the following command:
TF_ACC=1 go test "./huaweicloud/services/acceptance/servicestage" -v -coverprofile="./huaweicloud/services/acceptance/servicestage/servicestage_coverage.cov" -coverpkg="./huaweicloud/services/servicestage" -run TestAccRepoTokenAuth_basic -timeout 360m -parallel 10
=== RUN   TestAccRepoTokenAuth_basic
=== PAUSE TestAccRepoTokenAuth_basic
=== CONT  TestAccRepoTokenAuth_basic
--- PASS: TestAccRepoTokenAuth_basic (12.71s)
PASS
coverage: 3.9% of statements in ./huaweicloud/services/servicestage
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/servicestage      12.757s coverage: 3.9% of statements in ./huaweicloud/services/servicestage
```

File: docs/resources/servicestage_repo_token_authorization.md [hunk 1]
```diff
@@ -20,7 +20,6 @@ variable "personal_access_token"
 resource "huaweicloud_servicestage_repo_token_authorization" "test" {
   type  = "github"
   name  = var.authorization_name
-  host  = var.repository_host
```
Comment (deer-hang) [inline docs/resources/servicestage_repo_token_authorization.md:23]: remove variable `repository_host`.

Comment (Lance52259) [reply docs/resources/servicestage_repo_token_authorization.md:23]: done.

**Pull request merged**